flow/IThreadPoolTest.actor.cpp

```diff
@@ -11,7 +11,11 @@
 
 void forceLinkIThreadPoolTests() {}
 
+static ThreadReturnPromiseStream<std::string> notifications;
+
 struct ThreadNameReceiver final : IThreadPoolReceiver {
+	ThreadNameReceiver(const bool stream_signal = false) : stream_signal(stream_signal) {}
```

Comment:
nit: please use `FDB_DEFINE_BOOLEAN_PARAM` instead of a raw bool

Reply:
Thanks for the tip. Created a new ThreadPoolReceiver instead, according to the comment below.